### PR TITLE
Adds a small combustion turbine to Atmos maintenance.

### PR DESCRIPTION
**Pull request by furrycactus**
Repurposes a little bit of maintenance to give Atmospherics a singular turbine to play around with. Lockers/crates/maint loot that already existed in that room have been shifted to adjacent maintenance areas as to not disturb the overall balance of goodies.

The rationale behind this is that it'll give Atmos Techs something to do and to play with once thrusters are set up, while being completely optional. Messing around with it is never the expectation, hence the dim lighting, its tucked away nature, and how it totally lacks prior setup beyond necessary construction. (The SMES is empty and not upgraded, no canisters are in place and need to be manually found and filled elsewhere, all the pumps and computers are unconfigured, etc.)

Not really intended to replace the SM or INDRA, as combustion requires more maintenance and upkeep due to faster cooling, and actively burns fuel to run - plus it's only a single turbine. But it does give bored or curious Atmos Techs something else they can play with if they feel like it, once all the round-start stuff is done.

image

In conclusion:

ezgif-1-bbc246f85f

### CODE REVIEW

**Comment by furrycactus** [comment]
Augh I have a really weird map issue I'll just reopen this once I fix it.
